examples/qaoa.py

```diff
@@ -1,155 +1,71 @@
-"""Runs the Quantum Approximate Optimization Algorithm on Max-Cut.
```

Comment:
Is there a reason to remove the max cut example?

Reply:
Actually, it's still max-cut, but via a Boolean functions. I restored the comment that I shouldn't have deleted.